libpod/runtime.go

```diff
@@ -173,6 +173,24 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
 		runtime.state = state
 	}
 
+	// We now need to see if the system has restarted
+	// We check for the presence of a file in our tmp directory to verify this
+	runtimeAliveFile := filepath.Join(runtime.config.TmpDir, "alive")
```

Comment:
Just double checking.  I think this file is guaranteed to be wiped on a reboot correct?  If so, this looks good.

Reply:
As long as the user doesn't configure a path outside of tmpfs (mentioned explicitly as a no-no), it will definitely be wiped.

Reply:
👍 thanks for the confirmation